honeybee_openstudio/hvac/standards/thermal_zone.py

```diff
@@ -96,6 +96,7 @@ def thermal_zone_get_occupancy_schedule(model, thermal_zone):
     # Get all the occupancy schedules in spaces.
     # Check people added via the SpaceType and hard-assigned to the Space itself.
     occupancy_sch = None
+    num_people_sch = None
```

Comment:
Thank you, @pjennings!

@chriswmackey is the better person to review this, but I assume you meant to set `num_ppl_sch` to None.

```suggestion
    num_ppl_sch = None
```